build-system/pr-check.js

```diff
@@ -200,6 +200,15 @@ const command = {
   testBuildSystem: function() {
     execOrDie('npm run ava');
   },
+  testDocumentLinks: function(files) {
+    let docFiles = [];
+    files.forEach(file => {
+      if (isDocFile(file)) {
+        docFiles.push(file);
+      }
+    });
+    execOrDie(`${gulp} check-links --files ${docFiles.join(',')}`);
```

Comment:
We already import all the functions defined in `tasks/*`, we should be able to make this just be:
```
checkLinks(docFiles);
```

Unless there is something I am missing?